examples/basic/chat-search.py

```diff
@@ -88,7 +90,14 @@ def chat(opts: CLIOptions) -> None:
         vecdb=None,
     )
     agent = ChatAgent(config)
-    agent.enable_message(GoogleSearchTool)
+
+    if opts.provider == "google":
```

Comment:
@pchalasani shouldn't the `provider` flag be defined as `typer.Option` in the main parameters and update `opts = CLIOptions(model=model, provider=provider)` to give the user the ability to supply the value?

Reply:
yes `provider` should be exposed as an option in the `typer.Option`

Reply:
I see, thank you I will make this change.